sympy/stats/rv.py

```diff
@@ -457,6 +458,33 @@ def conditional_space(self, condition, normalize=True, **kwargs):
 
         return space(domain, density)
 
+class DependentPSpace(PSpace):
+
+    @staticmethod
+    def _create_MutlivariateNormalDistribution(syms, assumps):
+        from sympy.stats.symbolic_probability import Covariance, Variance
+        from sympy.stats.crv_types import Normal
+        assumps = assumps & And.fromiter([Eq(Variance(sym), sym.pspace.distribution.args[1])
+                                            for sym in syms])
+        xrep_dict = {eq.lhs: eq.rhs for eq in assumps.args}
+        mu = [sym.pspace.distribution.args[0] for sym in syms]
+        sigma = ImmutableMatrix(
+                [[Covariance(s1, s2).doit() for s2 in syms] for s1 in syms]
+                ).xreplace(xrep_dict)
+        return Normal('_', mu, sigma).pspace.distribution
+
+
+    @staticmethod
+    def compute_density(expr, assumps):
```

Comment:
this method refers to multivariate normal distributions without containing the name _Normal_. Maybe it can be confusing?

Reply:
Ah! I just forgot to mention that, I will use `getattr` to make a function call, like, if some `expr` has random variables of some X distribution then `_compute_MutlivariateXDistribution` would be called for construction of distribution. So, `compute_density` will be a general method making use of private constructors. 
Is it a good approach?

Reply:
Well... I would prefer to avoid `getattr` if possible. Consider the possibility of dispatching either on the class (`a.method`, `b.method`, ...) or with multipledispatch.